### PR TITLE
Add Globo Service

### PR DESCRIPTION
**Pull request by eduardoaddad**
Add Globoplay, Canais Globo, Premiere
Source for glbimg: https://s3.glbimg.com/v1/AUTH_c2986343db13479c82b8a9a301be7717/static/index.html

### CODE REVIEW

File: client/src/helpers/constants.js [hunk 1]
```diff
@@ -347,6 +347,11 @@ export const SERVICES = [
         id: 'youtube',
         name: 'YouTube',
     },
+
```
Comment (ainar-g) [inline client/src/helpers/constants.js:350]: Please remove this empty line.

File: internal/filtering/blocked.go [hunk 2]
```diff
@@ -374,6 +374,19 @@ var servicesData = []svc{{
...
+	rules: []string{
+		"||globoplay.globo.com^",
+		"||globoplay.com^",
+		"||globoplay.com.br^",
+		"||canaisglobo.globo.com^",
+		"||premiere.globo.com^",
+		"||canaisglobo.globo.com^",
+		"||glbimg.com^",
+		"||cloud-jarvis.globo.com^",
+		"||api-soccer.globosat.tv^",
```
Comment (ainar-g) [inline internal/filtering/blocked.go:388]: Please sort the rules.

Also, pay attention to the comment on `servicesData`.  `client/src/components/ui/Icons.js` must be updated as well.

**Review by ainar-g — CHANGES_REQUESTED** [review]
@ameshkov, is this service in our current service index?  Should we add it there?

Also, do you approve adding this service to AGH?

File: client/src/helpers/constants.js (hunk 1 above)
Comment (eduardoaddad) [reply client/src/helpers/constants.js:350]: done

**Review by eduardoaddad — COMMENTED** [review]
.

**Review by ainar-g — CHANGES_REQUESTED** [review]
It seems like the merge has gone wrong, since `constants.js` seems removed.

**Comment by eduardoaddad** [comment]
> It seems like the merge has gone wrong, since `constants.js` seems removed.

how can i restore this?

**Comment by eduardoaddad** [comment]
> It seems like the merge has gone wrong, since `constants.js` seems removed.

Correct?

**Review by ainar-g — CHANGES_REQUESTED** [review]
See previous comments

**Comment by ainar-g** [comment]
See https://github.com/AdguardTeam/AdGuardHome/issues/2604#issuecomment-1690454082.

**Pull request closed without merging**